README.md

```diff
@@ -11,7 +11,7 @@ jsonsubschema is very useful in analysing schema evolution and ensuring that new
 jsonsubschema also enables static type checking on different components of a system that uses JSON schema to describe data 
 interfaces among the system's different components.
 
-The details of JSON subschema are covered in our [**ISSTA 2021** paper](https://andrewhabib.org/publications/issta21-paper-JSONSubschema.pdf),
+The details of JSON subschema are covered in our [**ISSTA 2021** paper]([https://andrewhabib.org/publications/issta21-paper-JSONSubschema.pdf](https://software-lab.org/publications/issta2021_JSON.pdf)),
```

Comment:
It's strange to use a nested hyperlink here. Also, we might as well use an even-more-stable URL. How about the following?

```markdown
The details of JSON subschema are covered in our [**ISSTA 2021** paper](https://doi.org/10.1145/3460319.3464796),
```

Reply:
Whoops. I missed that error in the markdown. Initially I avoided the ACM URL because I failed to notice the publication was open access. I changed to the ACM link.